generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -162,17 +139,30 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting matching
        | null   |
        | "☠☠" |
 
-Scenario: Running a 'shorterThan' request alongside a non-contradicting matchingRegex constraint should be successful
-       And foo is shorter than 1
+@ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501